cmd/cmd.go

```diff
@@ -60,6 +60,17 @@ func ExecInCoreSnap() {
 		return
 	}
 
+	// can we re-exec? some distributions will need extra work before re-exec really works.
+	switch release.ReleaseInfo.ID {
+	case "fedora":
```

Comment:
nitpick^2: maybe `case "fedora", "centos", "rhel":` (in a single line) as this is still easy to read?